File: APICuponSmart/src/java/modelo/pojo/respuesta/RespuestaPromocion.java
package modelo.pojo.respuesta;

import java.util.List;
import modelo.pojo.entidad.Promocion;

public class RespuestaPromocion extends Mensaje{
    private List<Promocion> contenido;
    
    public List<Promocion> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Promocion> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaPromocion.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Promocion

class RespuestaPromocion(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Promocion>? = null
): Mensaje(error, mensaje)